### PR TITLE
Adding Database Service to the Catalog Application

### PR DESCRIPTION
**Pull request by xUrbana**

### CODE REVIEW

**Comment by ynigoreyes** [comment]
gotta fix your conflicts first

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (ynigoreyes) [inline .vscode/launch.json]: Where did you get this code from? You shouldn't have had to add any code to .vscode...

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (ynigoreyes) [inline .vscode/tasks.json]: Same with this one

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            },
+            "env": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            },
+            "sourceFileMap": {
+                "/Views": "${workspaceFolder}/Views"
+            }
+        },
+        {
+            "name": ".NET Core Attach",
```
Comment (ynigoreyes) [inline .vscode/launch.json]: Wait where is the process for running it as a service in Azure Dev Spaces?

**Review by ynigoreyes — CHANGES_REQUESTED** [review]
uhhhhhhh i'm not sure how this happened but how do you have so many changes in this branch? There are 64 changes and most of them are not even in your service?

File: .vscode/launch.json (hunk 1 above)
Comment (xUrbana) [reply .vscode/launch.json]: This was automatically added somehow, never touched this one.

File: .vscode/tasks.json (hunk 2 above)
Comment (xUrbana) [reply .vscode/tasks.json]: Never touched this one either.

File: src/CatalogApplication/Services/dbservice.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
+using ACMTTU.NoteSharing.Shared.SDK.Services;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+
+namespace ACMTTU.NoteSharing.Platform.CatalogApplication.Services
+{
+    public class ConnectionService : PlatformBaseService
```
Comment (ynigoreyes) [inline src/CatalogApplication/Services/dbservice.cs:8]: nit: Typically, the file name matches the class the file containers

**Pull request merged**